scripts/sidecarScriptApi.ts

```diff
@@ -103,12 +103,17 @@ export const launchProcess = (
 		procs[proc].stderr.on('data', (data: Buffer) => {
 			console.error(data.toString().trim());
 
-			if (resolverStartupErr && data.toString().includes(resolverStartupErr)) {
+			if (resolverJestErr && data.toString().trim().includes(resolverJestErr)) {
+				jestStatus = Failed;
+			}
+
+			if (resolverStartupErr && data.toString().trim().includes(resolverStartupErr)) {
 				resolve(Failed);
 			}
```

Comment:
Just want to understand this bit; if there's a startup error and `resolverStartupErr` is true then we bail immedaitely with Failed, but in the `resolverJestErr` case we just set it to failed and let the "close" handle it?